Tools/travis_depends.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -xe
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR/..
+git submodule update --init Classes/KIF &
+bundle install &
```

Comment:
to better control these concurrent tasks, you might use a function like:
```
function spawn_for_great_justice {
local run_this=$1 wait_for=${2:-500}
bash -c “$run_this” &
local dat_pid=$!
( sleep $wait_for ; if kill -0 $dat_pid ; then kill $dat_pid ; fi ) &
}
```
and then you know that you'll have some level of control over zombies... won't work all the time, but generally safer for small builds